src/dtm.jl

```diff
@@ -16,7 +16,17 @@ end
 #
 ##############################################################################
 
-# create col index lookup dictionary from a (sorted?) vector of terms
+"""
+```
+columnindices(terms)
+```
+
+Creates a column index lookup dictionary from a vector of terms.
+
+Parameters:
```

Comment:
The offline docs support limited markdown-type of formatting.
We could have the following change - `Parameters:` -> `# Arguments`